xml/README.md

```diff
@@ -4,3 +4,4 @@
 - [XML Libraries Support in Java](http://www.baeldung.com/java-xml-libraries)
 - [DOM parsing with Xerces](http://www.baeldung.com/java-xerces-dom-parsing)
 - [Write an org.w3.dom.Document to a File](https://www.baeldung.com/java-write-xml-document-file)
+- [Convert XML to HTML](https://www.baeldung.com/convert-xml-to-html-in-java)
```

Comment:
Let's leave this line out -- I don't know what the final URL will be, and I'll add the line soon after it's published.